release-guidelines.md

```diff
@@ -80,8 +80,6 @@ WSO2 Identity and Access Management team is pleased to announce the release of I
 
 You can download WSO2 Identity Server <code><em><RELEASED_VERSION> </em></code>from[ ](https://github.com/wso2/product-is/releases/download/v5.10.0-m2/wso2is-5.10.0-m2.zip)<code><em><IS_ARTIFACT_LOCATION></em></code>.
```

Comment:
lets remove this URL from the teamplate

Reply:
Fixed in https://github.com/wso2/product-is/pull/6740